api/src/main/java/org/itsallcode/openfasttrace/FilterSettings.java

```diff
@@ -24,208 +24,178 @@
-        {
-            return false;
-        }
-        if (!(obj instanceof FilterSettings))
-        {
-            return false;
-        }
-        final FilterSettings other = (FilterSettings) obj;
-        if (this.artifactTypes == null)
-        {
-            if (other.artifactTypes != null)
-            {
-                return false;
-            }
-        }
-        else if (!this.artifactTypes.equals(other.artifactTypes))
-        {
-            return false;
-        }
-        if (this.withoutTags != other.withoutTags)
-        {
-            return false;
-        }
-        if (this.tags == null)
-        {
-            if (other.tags != null)
-            {
-                return false;
-            }
-        }
-        else if (!this.tags.equals(other.tags))
-        {
-            return false;
-        }
-        return true;
-    }
-
-    /**
-     * Create filter settings that allow everything to pass unfiltered
-     * 
-     * @return <code>FilterSettings</code> that allow everything to pass
-     *         unfiltered
-     */
-    public static FilterSettings createAllowingEverything()
-    {
-        return new FilterSettings.Builder().build();
-    }
-
-    /**
-     * Builder for {@link FilterSettings}
-     */
-    public static class Builder
-    {
-        private Set<String> artifactTypes = Collections.emptySet();
-        private Set<String> tags = Collections.emptySet();
-        private boolean withoutTags = true;
-
-        /**
-         * Set the list of artifact types that the filter matches.
-         * 
-         * @param artifactTypes
-         *            artifact types that must be matched
-         * @return <code>this</code> for fluent programming
-         */
-        public Builder artifactTypes(final Set<String> artifactTypes)
-        {
-            this.artifactTypes = artifactTypes;
-            return this;
-        }
-
-        /**
-         * Set the list of tags that the filter matches.
-         * 
-         * @param tags
-         *            tags that must be matched
-         * @return <code>this</code> for fluent programming
-         */
-        public Builder tags(final Set<String> tags)
-        {
-            this.tags = tags;
-            return this;
-        }
-
-        /**
-         * Configure if filter allows items that have no tags.
-         * 
-         * @param noTags
-         *            <code>true</code> to match items without any tags
-         * @return <code>this</code> for fluent programming
-         */
-        public Builder withoutTags(final boolean noTags)
-        {
-            this.withoutTags = noTags;
-            return this;
-        }
-
-        /**
-         * Build an instance of type {@link FilterSettings}.
-         * 
-         * @return the new instance.
-         */
-        public FilterSettings build()
-        {
-            return new FilterSettings(this);
-        }
-    }
+public final class FilterSettings {
```

Comment:
Here something with the line ending formatter must have happended. This class should not be completely changed. Please check.